src/components/login/login.tsx

```diff
@@ -0,0 +1,111 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle, TextField } from '@material-ui/core';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+interface State {
+  APIClient: rest.Evergreen;
+  open : boolean;
+  submitted : boolean
+  username : string
+  password : string
+}
+
+class Props {
+}
+
+export class Login extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.handleClickOpen = this.handleClickOpen.bind(this);
+    this.handleClickClose = this.handleClickClose.bind(this);
+    this.handleClickSubmit = this.handleClickSubmit.bind(this);
+    this.state = {
+      APIClient: rest.EvergreenClient("admin", "e4f2c40463dcade5248d36434cb93bac", "http://localhost:8080/api"),
+      open : false,
+      submitted : false,
+      username : "",
+      password : "",
+    };
+  }
+
+  public handleClickOpen() : any{
```

Comment:
These functions here don't return anything, so they should be of type void (delete the :any return signature). Also in general functions shouldn't return "any" because we know the type they return

Reply:
Fixed- switched return type for all three handlers to void.